src/main/java/org/broadinstitute/hellbender/utils/IntervalUtils.java

```diff
@@ -289,17 +289,16 @@ else if ( new File(arg).exists() ) {
             final IntervalList il = IntervalList.fromFile(inputFile);
             isPicardInterval = true;
 
-            int nInvalidIntervals = 0;
             for (final Interval interval : il.getIntervals()) {
-                if ( glParser.isValidGenomeLoc(interval.getContig(), interval.getStart(), interval.getEnd(), true)) {
+                if (interval.getStart() - interval.getEnd() == 1 ) { // remove once a corrected version of the exome interval list is released.
+                    logger.warn("Possible incorrectly converted length 1 interval : " + interval);
```

Comment:
Also create a GATK4 ticket to remove this hack once a corrected version of the exome interval list is published.